package.json

```diff
@@ -62,7 +63,8 @@
     "prepare": "cd node_modules/typedoc && yarn install && yarn build",
     "lint": "yarn eslint .",
     "clean": "rm -rf dist/",
-    "build": "yarn tsc -p src/tsconfig.json",
+    "build": "yarn tsc -p src/tsconfig.json && yarn build:flowtypes",
```

Comment:
I added the `build:flowtypes` as part of the main build for this repository.

However, if you don't want to include the Flow files in the main NPM package for this repository, we can remove this line (it may be best since I assume you won't want to commit to making sure these Flow files work properly)

Reply:
I'm not against including the flowtypes in the final package, we can simply add a disclaimer in the README that flow types support is experimental and the package is intended for use with TS.

What I noticed though is that build:flowtypes takes significant time to run and the prepublish script is triggered after each invocation of `yarn install` which is a bit annoying. I found a package `in-publish` for that, but maybe you have some nicer solution to ensure that the commad is run only when publishing to npm

Also, I noticed that the command throws errors for bigint_like type:

```
Parsing txCertificate
$ /home/rafael/workplace/cardano/ledger/ledgerjs-cardano-shelley/node_modules/.bin/flowgen --add-flow-header --no-inexact ./dist/types/public.d.ts -o ./dist/types/public.js.flow
Error ---------------------------------------------- dist/types/public.d.ts:1:44


> 1 | export declare type bigint_like = number | bigint | string;
    |                                            ^^^^^^ Flow doesn't support BigInt proposal: https://github.com/facebook/flow/issues/6639
  2 | export declare enum AddressType {
  3 |     BASE = 0,
  4 |     POINTER = 4,


Parsing public
$ /home/rafael/workplace/cardano/ledger/ledgerjs-cardano-shelley/node_modules/.bin/flowgen --add-flow-header --no-inexact ./dist/types/internal.d.ts -o ./dist/types/internal.js.flow
Error ------------------------------------------- dist/types/internal.d.ts:16:38


  14 |     __type: 'uint64_t';
  15 | };
> 16 | export declare type _Uint64_bigint = bigint & {
     |                                      ^^^^^^ Flow doesn't support BigInt proposal: https://github.com/facebook/flow/issues/6639
  17 |     __type: 'uint64_t';
  18 | };
  19 | export declare type ValidBIP32Path = Array<Uint32_t> & {


Parsing internal
```
Aren't your types corrupted/incomplete as a result of that?

Reply:
`bigint` just gets replaced by `number` which works good enough.

Reply:
Okay, I split up the commands so it should only be building the flow types when publishing now.